x-pack/plugins/security_solution/public/resolver/view/assets.tsx

```diff
@@ -309,39 +298,52 @@ const SymbolsAndShapes = memo(() => (
       <path
         d="M87.52113,24.73352a3.7956,3.7956,0,0,0-1.43182-1.47166L45.91012.17918a3.8365,3.8365,0,0,0-3.82049,0L1.91029,23.26186A3.86312,3.86312,0,0,0-.00009,26.55445V72.7199a3.79834,3.79834,0,0,0,1.91041,3.29249L42.08963,99.09514a3.83689,3.83689,0,0,0,3.82049,0L86.08931,76.01239a3.79852,3.79852,0,0,0,1.91056-3.29249V26.55445A3.77643,3.77643,0,0,0,87.52113,24.73352Z"
         transform="translate(0.00013 0.39551)"
-        fill="#010101"
+        fill={isDarkMode ? '#010101' : '#fff'}
```

Comment:
:grey_question: My only reservation here is seeing dynamic properties on things that go into a `<defs>`. I have some vague recollection of 'things going wrong' with that just because it's an area a lot of Browser vendors don't plumb the depths of that far. I _think_ it's OK and it certainly _should_ work according to spec (and the only time you would possibly see anything wrong anway is when you switch the theme, right?). There are easy ways to fix it if we find a problem with it later, anyhow.